pgppubkey_to_jwk.py

```diff
@@ -1,17 +1,9 @@
-from authlib.common.encoding import int_to_base64
 import json
-import pgpy
+from josepy import JWK
 
+with open('public_release.pem', 'r') as f:
+    key = bytes(f.read(), 'utf-8')
 
-# Input is PGP armored pubkey
-key, _ = pgpy.PGPKey.from_file('key.asc')
```

Comment:
so `jwk.py` is for taking a PEM encoded key and dumping to JWK: basically this is used by the target `make test-key` in this repo which makes the corresponding JWK format for experiment/testing purposes only

the idea with `pgppubkey_to_jwk.py` however was to take an existing PGP RSA public key (i.e. one not PEM formatted) and dump it in the expected JWK format (the idea there was that I wanted to use our existing SecureDrop PGP release key for the ruleset channel, and the extension required the JWK)